src/imgui_tools.cpp

```diff
@@ -313,6 +313,10 @@ namespace wr::imgui::window
 					light.ang = light.ang / 180.f * 3.1415926535f;
 				}
 
+				static float light_size_temp = 0.0f;
+				ImGui::DragFloat("Light Size", &light_size_temp, 0.1f, 0.0f, 10.0f);
```

Comment:
bah (talked to you about it in person)

Reply:
Didn't remove the light_size_temp, but it made me realize I had to convert it to radians instead of that hacky "map it back". Now an angular extent of 0.5 degrees produces shadows that are consistent with the ones from our sun.